python/cloudtik/runtime/spark/scripts/services.sh

```diff
@@ -21,7 +21,8 @@ start-head)
 stop-head)
     $HADOOP_HOME/bin/yarn --daemon stop resourcemanager
     $SPARK_HOME/sbin/stop-history-server.sh
-    jupyter lab stop
+    # workaround for stopping jupyter when password being set
+    kill $(pgrep jupyter)
```

Comment:
If there is no jupyter server process, kill will missing pid parameter and show error:

kill: usage: kill [-s sigspec | -n signum | -sigspec] pid | jobspec ... or kill -l [sigspec]

Reply:
Thanks for the review. Refined the scripts to stop running jupyter.